lib/event/src/lib.rs

```diff
@@ -11,14 +14,16 @@ use std::{collections::BTreeMap, iter::FromIterator};
 use string_cache::DefaultAtom as Atom;
 
 pub mod discriminant;
+#[cfg(feature = "lua")]
```

Comment:
I think if we commit to doing this we should consider our feature flags at the same time. THis is a good chance to create some harmony and structure between the two!

Reply:
Agreed, but this will need some deliberation. I didn't really want to embed the knowledge that this should be turned on by `transforms-lua` into the lib, but I also don't want to get rid of the `transforms-*` feature scheme at the top level.